nilearn/maskers/multi_nifti_masker.py

```diff
@@ -63,7 +62,7 @@ def _get_mask_strategy(strategy):
 
 
 @fill_doc
-class MultiNiftiMasker(NiftiMasker, CacheMixin):
+class MultiNiftiMasker(NiftiMasker):
```

Comment:
showing the class inheritance tree showed that this inheritance was not necessary as MultiNiftiMasker already inherits from NiftiMasker